src/assets/css/main.css

```diff
@@ -0,0 +1,1911 @@
+/*styles for home content only*/
+.clearfix:after,.clearfix:before {
```

Comment:
It looks like this is the built css file from the prototype. The prototype contains the raw scss files, so we'll need to incorporate those rather than this built file. This can be done as part of the fixup css issue that you're still having trouble with, but wanted to note this here.